lib/jsdom/living/register-elements.js

```diff
@@ -7,7 +7,58 @@ const mappings = {
   "http://www.w3.org/1999/xhtml": {
     HTMLElement: {
       file: require("./generated/HTMLElement.js"),
-      tags: []
+      tags: [
```

Comment:
Is there a full list of these centralized somewhere, or are we picking these through the whole HTML spec?

Reply:
Ah, there's an [elements](https://html.spec.whatwg.org/#elements-3) section which has them all.